src/functions/any.js

```diff
@@ -21,6 +21,10 @@ export const any = wrap({
             elements were truthy. The function returns @false otherwise, or
             if the sequence was empty.
         `),
+        throws: (`
+            The function throws a @NotBoundedError when the input sequence was
+            not known to be bounded.
+        `),
```

Comment:
All functions that may throw an error in their `implementation` function should describe the errors and the circumstances under which they are thrown in `throws` strings like this